code/game/gamemodes/miniantags/borer/borer.dm

```diff
@@ -764,7 +764,9 @@ GLOBAL_VAR_INIT(total_borer_hosts_needed, 10)
 		if("Blindness")
 			victim.blind_eyes(2)
 		if("Deafness")
-			victim.ear_deaf = 20
+			var/obj/item/organ/ears/ears = victim.getorganslot("ears")
+			if(istype(ears))
+				ears.deaf = max(ears.deaf, 20)
```

Comment:
And again
Why you use adjustEarDamage proc in some cases and direct manipulations in other cases?